### PR TITLE
Request/add missing day

### PR DESCRIPTION
**Pull request by khushbuSatasiya**
add missing day and work from home

### CODE REVIEW

File: src/features/request/components/addMissingDay.tsx [hunk 1]
```diff
@@ -0,0 +1,284 @@
...
+
+		if (action === 'inTime' && formattedTime.includes(':') && formattedTime.length === 5) {
+			inTimeArr = formattedTime.split(':');
+			tmpInTime = new Date().setHours(Number(inTimeArr[0]), Number(inTimeArr[1]));
+			inTimeArr.length === 2 && setInTime(tmpInTime);
+		} else if (action === 'outTime' && formattedTime.includes(':') && formattedTime.length === 5) {
+			outTimeArr = formattedTime.split(':');
+			tmpOutTime = new Date().setHours(Number(outTimeArr[0]), Number(outTimeArr[1]));
+			outTimeArr.length === 2 && setOutTime(tmpOutTime);
+		}
```
Comment (daxesh-parmar-scaletech) [inline src/features/request/components/addMissingDay.tsx:76]: please optimize this condition

File: src/features/request/components/addMissingDayConfirmModal.tsx [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+	onClose: () => void;
+}
+
+const AddMissingDayConfirmModal: FC<IProps> = ({ isConfirm, setIsConfirm }) => {
+	return (
+		<>
+			<Modal
+				size='sm'
+				opened={isConfirm}
+				onClose={() => console.log()}
```
Comment (daxesh-parmar-scaletech) [inline src/features/request/components/addMissingDayConfirmModal.tsx]: add onClose function from props

File: src/features/request/components/addMissingDayConfirmModal.tsx [hunk 3]
```diff
@@ -0,0 +1,51 @@
+import React, { FC } from 'react';
+
+import { Button, Flex, Modal, Paper, Text } from '@mantine/core';
+import { IconAlertTriangle } from '@tabler/icons-react';
+
+interface IProps {
+	isConfirm: boolean;
+	setIsConfirm: (action: boolean) => void;
+	isSuccess: boolean;
```
Comment (daxesh-parmar-scaletech) [inline src/features/request/components/addMissingDayConfirmModal.tsx]: remove `isSuccess` if not needed

File: src/features/request/components/workFromHome.tsx [hunk 4]
```diff
@@ -0,0 +1,75 @@
...
+		await httpService
+			.post(API_CONFIG.path.wfh, payload)
+			.then((res) => {
+				setIsLoading(false);
+				setIsSuccess(true);
+
+				setTimeout(() => {
+					onClose();
+				}, 3000);
+				// showNotification(res, theme.colors.blue[6], theme.colors.blue[6]);
```
Comment (daxesh-parmar-scaletech) [inline src/features/request/components/workFromHome.tsx]: remove commented code

File: src/features/request/components/workFromHomeModal.tsx [hunk 5]
```diff
@@ -0,0 +1,210 @@
...
+									/>
+
+									<TextInput
+										placeholder={'Write a proper reason'}
+										label='Reason'
+										w={'100%'}
+										mt={24}
+										// onChange={handleChange('isConfirm')}
+										// onBlur={handleBlur('isConfirm')}
+										{...form.getInputProps('reason')}
```
Comment (daxesh-parmar-scaletech) [inline src/features/request/components/workFromHomeModal.tsx:126]: remove commented code

**Review by daxesh-parmar-scaletech — CHANGES_REQUESTED** [review]

File: src/features/request/components/workFromHomeModal.tsx (hunk 5 above)
Comment (khushbuSatasiya) [reply src/features/request/components/workFromHomeModal.tsx:126]: done

File: src/features/request/components/workFromHome.tsx (hunk 4 above)
Comment (khushbuSatasiya) [reply src/features/request/components/workFromHome.tsx]: done

File: src/features/request/components/addMissingDayConfirmModal.tsx (hunk 3 above)
Comment (khushbuSatasiya) [reply src/features/request/components/addMissingDayConfirmModal.tsx]: done

File: src/features/request/components/addMissingDayConfirmModal.tsx (hunk 2 above)
Comment (khushbuSatasiya) [reply src/features/request/components/addMissingDayConfirmModal.tsx]: done

File: src/features/request/components/addMissingDay.tsx (hunk 1 above)
Comment (khushbuSatasiya) [reply src/features/request/components/addMissingDay.tsx:76]: done

**Review by daxesh-parmar-scaletech — APPROVED** [review]

**Pull request merged**